core/payment/src/processor.rs

```diff
@@ -245,6 +246,7 @@ impl PaymentProcessor {
 
         // TODO: Implement re-sending mechanism in case SendPayment fails
 
+        counter!("payment.invoices.requestor.paid", 1);
```

Comment:
This is not semantically accurate. Single payment can be used to pay off multiple invoices if they have the same payee address. (We don't do that now, but data model allows it.) So it should be rather 'sent payments' not 'paid invoices'. If you really want paid invoices then you have to look at the length of `agreement_payments` vector here.

Reply:
Thanks for noticing this.